.eslintrc.json

```diff
@@ -22,10 +25,32 @@
   ],
 
   "rules": {
-    "comma-dangle": ["error", "only-multiline"],
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "linebreak-style": ["error", "unix"],
-    "quotes": ["error", "single"],
-    "semi": ["error", "always"]
+    "indent": [
+      2, 
+      4
+    ],
+    "comma-dangle": [
+        "error", 
+        "never"
+    ],
+    "no-unused-vars": [
+        "error", 
+        {
+            "vars": "local", 
+            "args": "none" 
+        }
+    ],
+    "arrow-parens": [
+        2, 
+        "as-needed"
+    ],
+    "no-shadow": "off",
+    "consistent-return": "off",
+    "prefer-promise-reject-errors": "off",
+    "arrow-body-style": "off",
+    "import/no-named-as-default-member": "off",
```

Comment:
Why do you not want this rule? Seems like it does everything to let us know where our imports might be defined and keeps our import style concise throughout the project.